splitio/push/segmentworker.py

```diff
@@ -11,7 +11,7 @@ class SegmentWorker(object):
 
     _centinel = object()
 
-    def __init__(self, synchronize_segment, segment_queue):
+    def __init__(self, synchronize_segment, segment_queue, telemetry_runtime_producer):
```

Comment:
why is this change? you are not using telemetry here.

Reply:
ah yes, will remove